nucypher/utilities/prometheus/collector.py

```diff
@@ -121,6 +124,7 @@ def _collect_internal(self) -> None:
 
         self.metrics["learning_status"].state('running' if self.ursula._learning_task.running else 'stopped')
         self.metrics["known_nodes_gauge"].set(len(self.ursula.known_nodes))
+        self.metrics["availability_score_gauge"].set(self.ursula._availability_tracker.score)
```

Comment:
I'm pretty sure this is the correct reference according to https://github.com/nucypher/nucypher/blob/main/tests/acceptance/network/test_availability.py#L41:

```
assert ursula._availability_tracker.score == 10
```

Hoping a reviewer can verify that this will export the correct availability score.

Reply:
Interesting situation here - availability tracker can be disabled (see https://github.com/nucypher/nucypher/blob/main/nucypher/characters/lawful.py#L1214) in which case it is never started and it simply reports the initial base score of `10`. In this case this code will report `10` as opposed to something indicating that it is disabled - which can lead to confusion.

It may be better to store something like `-1` if the `Gauge` allows for it, or store some other type of metric for the score that allows for text eg. `NA`, or `disabled` to indicate when the tracker is not being used.

Reply:
Good point, I was wondering about this. Currently running my ursulas with `--disable-availability-check`, so this scenario would apply for me.

Biased from my own use case, but I'd prefer `-1` as the output if the tracker is disabled - if only to work well with prometheus queries downstream that are expecting numbers.

Reply:
Updated with https://github.com/nucypher/nucypher/pull/2264/commits/894b69678276a0dbb78e8b0876edd0dcd86ce4dc